harness/go/harness/util.go

```diff
@@ -41,6 +41,29 @@ func FindEvent(history client.HistoryEventIterator, cond func(*historypb.History
 	return nil, nil
 }
 
+// GetWorkflowResultPayload returns the first payload of a workflow result
+func GetWorkflowResultPayload(ctx context.Context, client client.Client,
+	workflowId string) (*common.Payload, error) {
```

Comment:
```suggestion
func GetWorkflowResultPayload(
	ctx context.Context,
	client client.Client,
	workflowID string,
) (*common.Payload, error) {
```

If you must wrap, this is usually the better way. And "ID" in Go is usually capitalized (when not part of generated code).

Reply:
Much cleaner. Is there are guideline for max columns?, I'm assuming about 120...

Reply:
No strict guideline in Go, but yeah 120 is reasonable (assuming tab width at 2, different people have different widths)